### PR TITLE
Update dlp-policy-tips-reference.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated  a note on Policy Tips in older versions of Outlook

Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9393

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/dlp-policy-tips-reference.md [hunk 1]
```diff
@@ -169,7 +169,10 @@ The list of out-of-the-box sensitive information types that will be detected for
 - U.S. Individual Taxpayer Identification Number (ITIN)
 - U.S. Social Security Number (SSN)
 
-Please note that custom sensitive information types are also supported for DLP policy tips in addition to the above out-of-the-box sensitive information types.
+Please note that some custom sensitive information types are also supported for DLP policy tips in addition to the above out-of-the-box sensitive information types.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-policy-tips-reference.md]:
```suggestion
Note that some custom sensitive information types are also supported for DLP policy tips in addition to the above out-of-the-box sensitive information types.
```

File: microsoft-365/compliance/dlp-policy-tips-reference.md [hunk 2]
```diff
@@ -169,7 +169,10 @@ The list of out-of-the-box sensitive information types that will be detected for
...
+
+> [!NOTE]
+> Not all elements of custom sensitive information types are compatible with every version of Office. Entity elements for Custom SITs, such as Functions may cause incompatibility.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-policy-tips-reference.md]:
```suggestion
> Not all elements of custom sensitive information types are compatible with every version of Office. Entity elements for Custom SITs, such as Functions, may cause incompatibility.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Comment by chrfox** [comment]
reviewing with engineering

**Pull request merged**